modules/check_in/app/controllers/check_in/v2/pre_check_in_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module CheckIn
+  module V2
+    class PreCheckInController < CheckIn::ApplicationController
+      def show
+        head :not_implemented
+      end
+
+      def create
+        head :not_implemented
+      end
+
+      private
+
+      def session_params
```

Comment:
Suggest changing this to pre_check_in_params to align with controller name

Reply:
Sure, modified the method name.